R/03-modelResults3D.R

```diff
@@ -313,6 +313,9 @@ modelResults3DUI <- function(id, title = ""){
           radioButtons(inputId = ns("mapType"), label = "Plot type", inline = TRUE,
                        choices = c("Map", "Time course"),
                        selected = "Map"),
+          numericInput(inputId = ns("scatterDecPlace"),
+                     label = "Input decimal places for scatter plot",
+                     min = 0, max = 10, value = 2, step = 1, width = "100%"),
         conditionalPanel(
```

Comment:
Here it would be great to have a conditional panel so that we only see the new input if
 `"input.mapType == 'Time course'"`.